Misc/NEWS.d/next/Library/2021-02-06-05-34-01.bpo-43048.yCPUmo.rst

```diff
@@ -0,0 +1 @@
+When traceback is formatting a RecursionError with a long __context__ chain it runs the risk of itself raising a RecursionError. This patch catches the corresponding recursive call's RecursionError, truncates the traceback and emits a warning instead.
```

Comment:
This is ambiguous because there is also traceback formatting code in the interpreter. Also probably too long, and mentions context but not cause.  This is probably enough to indicate the impact (and there are always more details in the bpo):

Handle `RecursionError`s in :class:`~traceback.TracebackException`'s constructor, so that long exceptions chains are truncated instead of causing traceback formatting to fail.

Reply:
done. thanks for the patience!